_layouts/homepage.html

```diff
@@ -94,6 +94,7 @@ <h3 class="mt-3">Deploy</h3>
           <ul>
             <li><a href="{{ '/cockroachcloud/create-your-cluster.html' | relative_url }}">CockroachDB Cloud Deployment</a></li>
             <li><a href="{{ '/stable/orchestration.html' | relative_url }}">Self-Managed Deployment</a></li>
+            <li><a href="{{ '/stable/choose-a-deployment-option.html' | relative_url }}">Choose a Deployment Option</a></li>
             <li><a href="{{ '/stable/recommended-production-settings.html' | relative_url }}">Production Checklist</a></li>
             <li><a href="{{ '/stable/performance.html' | relative_url }}">Performance Profiles</a></li>
             <li><a href="{{ '/stable/upgrade-cockroach-version.html' | relative_url }}">Cluster Maintenance</a></li>
```

Comment:
Should we remove one of the old links? Not sure if it matters but the other sections have 5 or less links

Reply:
If it were up to me, I'd remove Performance Profiles and I renamed Cluster Maintenance to Upgrade CockroachDB. What do you think?